lib/connectors_sdk/office365/custom_client.rb

```diff
@@ -75,12 +75,11 @@ def share_point_drives(fields: [])
         # recently created groups (new Private Team site will be there) to reduce friction and index this site
         # earlier.
         # See: https://github.com/elastic/ent-search/pull/3581
-        share_point_sites = (sites(:fields => %w[id]) + recent_share_point_group_sites(:fields => %[id]))
+        share_point_sites = (sites(:fields => %w[id,name]) + recent_share_point_group_sites(:fields => %w[id,name]))
```

Comment:
:( based off of the failing specs [here](https://github.com/elastic/ent-search/pull/6521), I think this will require you to re-record the document lifecycle specs :(

Reply:
Yes, as I mentioned here https://github.com/elastic/ent-search/pull/6521#issuecomment-1120679161, will re-record after this is merged, to avoid any further changes.